backup_and_restore/application_backup_and_restore/troubleshooting/migration-using-must-gather-doc.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc
+// * migration_toolkit_for_containers/troubleshooting-mtc.adoc
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="migration-using-must-gather-doc"]
+= Using the must-gather tool
+
+You can collect logs, metrics, and information about {local-product} custom resources by using the `must-gather` tool.
```

Comment:
In the preview I see:
{local-product}
Should it be converted to some other value?
Currently it's not clear and seems like a typo.